### PR TITLE
Add veto files line to smb.conf snippet

### PR DESCRIPTION
**Pull request by marians**
This helps avoid creation of extra files when connecting from MacOS client via SMB.

The extra files make scanning directories slower and are just noise when browsing the directories using anything else than the Mac finder.

### CODE REVIEW

File: docs/INSTALL-stretch.md [hunk 1]
```diff
@@ -187,14 +187,15 @@ Now add the specific folder that we want to be exposed to the home network in th
 
 ~~~~
 [pi_jukebox]
-   comment= Pi Jukebox
+   comment=Pi Jukebox
```
Comment (marians) [inline docs/INSTALL-stretch.md]: This is unrelated, just removing a blank for consistency

**Comment by MiczFlor** [comment]
Hi @marians 
 
nice catch! Thanks for the contribution and keep it coming!
